ProjectSidewalk/SidewalkWebpage
# Clustering is now done on sidewalk-mturk-v2

I have integrated the python script that clusters ground truth labels, along with a simple GUI for running the script on a route. I also wrote a query to get all the label data for a clustering session, for use in the GT resolution tool.

To test the code:
1. Make sure that you are signed out of PS on localhost (o/w mturk branch won't work correctly RN). Even though there is nothing on the navbar, you should be able to go to localhost:9000/signOut and that should sign you out. Alternatively, you can test in incognito mode :upside_down_face: 
2. Run create_routes.py and hit_creation_sim.py
3. Audit the same route 3 times, as a different user w/ different assignment id each time. URLs would look like...
`localhost:9000/?assignmentId=1&workerId=M1&hitId=ROUTE&turkSubmitTo=localhost:5000/post`
`localhost:9000/?assignmentId=2&workerId=M2&hitId=ROUTE&turkSubmitTo=localhost:5000/post`
`localhost:9000/?assignmentId=3&workerId=M3&hitId=ROUTE&turkSubmitTo=localhost:5000/post`
4. Got to `localhost:9000/clustering`, enter the id of the route you just audited into the text box. Then click the `Run Clustering` button. (now the clustering* tables should be filled in)
5. To test the query for the label data to be used in the GT resolution tool, go to `localhost:9000/labelsForGtResolution/<SESSION_ID>`, filling in your session id (it is probably 1).

Fixes #877
